app/views/local_groups/show.html.slim

```diff
@@ -26,3 +26,8 @@
       strong Mailtrain List ID
       br
       = @local_group.mailtrain_list_id.present? ? @local_group.mailtrain_list_id : "-"
+
+    p
+      strong Welcome Message
+      br
+      = @local_group.welcome_email_body.body.html_safe unless @local_group.nil?
```

Comment:
`@local_group` will never be nil on a show page. I would instead write:

`@local_group.welcome_email_body&.body&.present? ? @local_group.welcome_email_body.body. : "-"`

This may not be perfect but should be fine. Test it with a local group having a welcome message and one not having a welcome message.